### PR TITLE
Fix crash in tpath

### PR DESCRIPTION
**Pull request by howardjohn**
I have no idea what this code does, so this may be a bad fix...

### CODE REVIEW

**Comment by jacob-delgado** [comment]
cc @howardjohn do you want approval? Did you find this issue due to the fuzzer?

**Comment by howardjohn** [comment]
Would be good for someone that understands this code to review it. This was from fuzzer

File: operator/pkg/tpath/tree.go [hunk 1]
```diff
@@ -337,6 +337,10 @@ func setPathContext(nc *PathContext, value interface{}, merge bool) error {
 func setValueContext(nc *PathContext, value interface{}, merge bool) (bool, error) {
 	vv, mapFromString := tryToUnmarshalStringToYAML(value)
 
+	if nc.Parent == nil {
+		return false, nil
```
Comment (richardwxn) [inline operator/pkg/tpath/tree.go:341]: move to the beginning?

**Review by richardwxn — APPROVED** [review]
feel free to remove label and merge after addressing the comment

**Pull request merged**